contracts/DCAHubCompanion/DCAHubCompanionMulticallHandler.sol

```diff
@@ -19,6 +19,23 @@ abstract contract DCAHubCompanionMulticallHandler is Multicall, DCAHubCompanionP
     permissionManager.permissionPermit(_permissions, _tokenId, _deadline, _v, _r, _s);
   }
 
+  function depositProxy(
+    address _from,
+    address _to,
+    uint256 _amount,
+    uint32 _amountOfSwaps,
+    uint32 _swapInterval,
+    address _owner,
+    IDCAPermissionManager.PermissionSet[] calldata _permissions,
+    bool _transferFromCaller
+  ) external returns (uint256 _positionId) {
+    if (_transferFromCaller) {
```

Comment:
This is here because if we are using balance that was withdrawn on the same multicall, we already have the balance in the contract 🔥